app/src/main/report/actions/reportsActions.ts

```diff
@@ -17,7 +20,38 @@ export const reportsActions = {
         }
     },
 
+    setCurrentReport(report: string) {
+        return {
+            type: ReportTypeKeys.SET_CURRENT_REPORT,
+            data: report
+        }
+    },
+
+    setCurrentVersion(version: string) {
```

Comment:
I'm wondering if we can do away with `currentVersion` altogether now - the reason we needed it before is because of the way we cached version details in the store - we had a lookup from version ids to version details, and when `currentVersion` changed we looked up the current details accordingly. But since we're moving away from that, I think we can always just get the current version id from `versionDetails.id`...